public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -232,4 +245,11 @@ export class EhrViewComponent implements OnInit, OnDestroy {
     concepts.forEach(concept => concept.expanded = false);
     r.expanded = true;
   }
+  public toggleTopConcepts() {
+    if (!this.showTopConcepts) {
```

Comment:
nit: `this.showTopConcepts = !this.showTopConcepts;`

Reply:
Addressed. Thanks.